### DISCUSSION TITLE
SetupListOperation

### DISCUSSION THREAD

**Discussion by KPF-Engenharia** [Laravel-Backpack/community-forum#882] [General]
Hello guys, 

every time you access the route backpack_url('user') the route with ?role_id=3 appears. How can only /users appear? detail: all users appear and not role 3 users

Image:
https://ibb.co/sHcVzSn

**Comment by o15a3d4l11s2** [comment]
Check the generated URL for the user CRUD. Depending on the version of Backpack that you have, you can check in `menu_items.blade.php` or `sidebar_content.blade.php`. You should have an item like
```php
<x-backpack::menu-dropdown-item title="Users" icon="la la-user" :link="backpack_url('user')" />
```
or
```php
<li class="nav-item"><a class="nav-link" href="{{ backpack_url('user') }}"><i class="nav-icon la la-user"></i>Users</a></li>
```
If it contains something different, then this could be the issue.

**Reply by KPF-Engenharia** [reply]
dd($this->crud) 
https://ibb.co/Nj1jSQQ

```
 <li class="nav-item"><a class="nav-link" href="{{ backpack_url('user') }}"><i class="nav-icon la la-users"></i> 
       <span>Usuários</span></a>
 </li>
```

**Reply by KPF-Engenharia** [reply]
In hidden navigation mode, I access the /user route and it is correct, however, when accessing the /user?role=2 route only once. This access stays forever when I access backpack_url('user')

**Reply by o15a3d4l11s2** [reply]
Ah, ok. This is expected behaviour. If you set a specific value to a filter (in your case the `role` filter), then the filters are persisted and even if you manually enter `/admin/user`, it will still show preset next time you visit the CRUD. I am not sure how you can force-clear them on the menu item click but still keep them whenever user is going through the CRUD page records.

**Reply by KPF-Engenharia** [reply]
I need this not to happen, I'm having problems with other entities in the same way

**Reply by KPF-Engenharia** [reply]
I created this http://localhost:8989/user?perfil=all, when it is all it shows everything otherwise it shows the id, but doing this for everything similar is annoying and the route does not look pleasant to the user

**Comment by jcastroa87** [comment]
Hello @KPF-Engenharia, as @o15a3d4l11s2 says, this is the expected behavior if you do not want to use a persistent table, you need to disable it in /config/backpack/operations/list.php

```
'persistentTable' => false,
```

Or you can follow our [documentation](https://backpackforlaravel.com/docs/6.x/crud-cheat-sheet#persistent-table) to handle in controller

Cheers.

**Reply by KPF-Engenharia** [reply]
@jcastroa87 thanks man .. problem solved

**Comment by jcastroa87** [comment]
Great, you solve it, I will close the issue, but feel free to re-open or create a new one if needed.

Cheers.
